src/main/src/main/java/org/geoserver/catalog/WMSLayerInfo.java

```diff
@@ -55,5 +55,9 @@
 
     public StyleInfo getDefaultStyle();
 
+    public boolean isRespectMetadataBBox();
```

Comment:
Just a quick thing, can you make this "english readable"? IsMetadataBBoxRespected()?

Reply:
sure

Reply:
@aaime requested changes applied